_drafts/SC3-1-1-html-has-lang.md

```diff
@@ -0,0 +1,42 @@
+---
+name: SC3-1-1-html-has-lang
+
+description:
+  This rule checks that the `html` element has a `lang` or `xml:lang` attribute.
+
+success_criterion:
+- 3.1.1
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Annika Nietzio
+- Jey Nandakumar
+---
+
+## Test Procedure
+
+### Applicability
+
+The root element of the page, if it is an `html` element.
+
+### Expectation
+
+The test target has a `lang` or `xml:lang` attribute.
+
```

Comment:
I think `xml:lang` should only count as a pass if this is an XHTML document - i.e. one served with a MIME type of application/xhtml+xml, application/xml or text/xml.

Reply:
@jribbens To the best of my knowledge assistive technologies don't care about the MIME type of a page. Is your concern regarding accessibility support, or is it a standards question?

Reply:
Well, both - if it's HTML then according to the specification having only `xml:lang` is the same as not having the language identified at all, and I was under the impression that assistive technologies ignore `xml:lang` anyway.